includes/lib/transifex-live-integration-rewrite.php

```diff
@@ -162,21 +162,26 @@ static function reverse_hard_link( $lang, $link, $languages_map, $source_lang,
 		if ( empty( $lang ) || empty( $languages_map ) ) {
 			return $link;
 		}
-                elseif ( !in_array( $lang, array_values( $languages_map ) ) || $source_lang == $lang ) {
+		elseif ( !in_array( $lang, array_values( $languages_map ) ) || $source_lang == $lang ) {
 			return $link;
 		}
 
 		preg_match( $pattern, $link, $m );
 		if ( count( $m ) > 1 ) {
 			$link = str_replace( $m[1], $lang, $m[0] );
 		} else {
-			/* Check if the path starts with the language code,
-			 * otherwise prepend it. */
-			$parsed = parse_url( $link );
-			if ( substr($parsed['path'], 1, strlen($lang))  != $lang ) {
-				$parsed['path'] = '/' . $lang . $parsed['path'];
+			$site_host = parse_url(site_url())['host'];
+			$link_host = parse_url($link)['host'];
```

Comment:
If we use `site_url()` to take the hostname how will this behaves with links without it? (ex. `/get-in-touch`).

Reply:
Thx for pointing out!

The plugin does NOT support URL rewritting of partial links (without a host part). However since we can't obviously disallow a user to insert such a link, we added code to assign default values for the 'host' part in order to fix possible E_NOTICE errors.